ohsome/response.py

```diff
@@ -87,7 +87,15 @@ def _as_geodataframe(
         """
 
         if len(self.data["features"]) == 0:
-            return gpd.GeoDataFrame(crs="epsg:4326", columns=["@osmId", "geometry"])
+            return gpd.GeoDataFrame(
+                crs="epsg:4326",
+                columns=["@osmId", "geometry"]
+                + (
+                    list(explode_tags) + ["@other_tags"]
+                    if explode_tags is not None
```

Comment:
Explode tags can be None. In that case the functionality of Version 0.2 is restored returning a df of maximal extent:

https://github.com/GIScience/ohsome-py/pull/165/files#diff-5ca71800389e412d6d50d3a8f5591ebadb75b64ecb2550a921e82afa67d1d66fR33

> :param explode_tags: By default, tags of extracted features are stored in a single dict-column. You can specify
>        a tuple of tags that should be popped from this column. To disable it completely, pass None. Yet, be aware that
>        you may get a large but sparse data frame.

I removed the outdated documentation from the private functions (and added typing)

> As a minor comment: If None check is needed, I would do it in a separated line instead of all in one statement.

The line you commented is a ternary operator that depends on the state of `explode_tags`. One can argue about the readability of ternary operators (and we can change to a regular if clause), but having a single central None check seems difficult because I use that decision in different places.

Reply:
Okay, thanks for the throughout explanation